src/styles/breakpoints.css

```diff
@@ -0,0 +1,32 @@
+$Breakpoint-mobile: 986px;
```

Comment:
Can we use either `snake_case` or `kebab-case`?

Reply:
✅